tox.ini

```diff
@@ -1,22 +1,27 @@
 [tox]
-envlist = 
-    py26,py27,py32,py33,pypy,cover
+envlist =
+    py27,py36,py37,py38,py39,pypy27,pypy3,coverage
```

Comment:
I think we might need to run coverage on both py2 and py3, then combine them, like we do for Pyramid 1.10. This assumes there are different code paths depending on Python version. I haven't checked, but we could check by running coverage manually on py2 then py3.

Reply:
This setup effectively runs it across all versions and combines it when run through `tox`. In GHA we combine 27 and 39 with `tox -e py27,py39,coverage`.

As it goes, I think the 39 coverage is a strict superset of 27 at the moment, but of course that could change.

Reply:
I don't think that's true. Please have a look at https://github.com/Pylons/pyramid/blob/1.10-branch/tox.ini

Reply:
@stevepiercy https://github.com/Pylons/pyramid_layout/pull/37/files#diff-03609cb60b0c6e92fb771eb8787d6722b8c31ca4c03eabc788e147acd8c6fb43R54

This does run coverage for both here, and the `tox.ini` shipped here runs coverage for all versions and environments, not just the one named py2-cover or py3-cover. That is why there is no extra testenvs necessary in this case.

Reply:
OK, I did not think to look over yonder: https://github.com/jon-betts/pyramid_layout/runs/2554929471?check_suite_focus=true#step:5:64

This is good.